packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -146,13 +163,19 @@ export const Collection = <T,>(props: CollectionProps<T>) => {
     const {classes, cx} = useStyles(null, {classNames, name: 'Collection', styles, unstyled});
     const collectionID = useId('dnd-droppable');
 
-    const [values, {append, remove, reorder}] = useControlledList({value, onChange, defaultValue});
-    const hasOnlyOneItem = values.length === 1;
+    const hasOnlyOneItem = value.length === 1;
     const removeItem = (index: number) => () => {
-        remove(index);
+        removeListItem(index);
         onRemoveItem?.(index);
```

Comment:
Since this prop already exist, maybe we should reuse it instead of having `onRemoveItem` and `removeListItem`. I think we should use either 
- `reorderListItem`
- `removeListItem`
- `insertListItem`
or
- `onRemoveItem`
- `onInsertItem`
- `onReorderItem`
but not a mix of the 2 naming patterns

Thoughts? I personally have a preference for the `onSomething` pattern. It sounds more like the `onChange` that we had.

Reply:
agreed. I'll leverage the already existing `onRemoveItem`